WORKSPACE

```diff
@@ -68,6 +68,14 @@ http_file(
     ],
 )
 
+http_file(
+    name = "libvirt_libs_aarch64",
+    sha256 = "b660f39790b4d14f47e921d9f11d4e6d020f75de5f7262bf8ddd1daf6339b741",
+    urls = [
+        "https://dl.fedoraproject.org/pub/fedora/linux/releases/32/Everything/aarch64/os/Packages/l/libvirt-libs-6.1.0-2.fc32.aarch64.rpm",
```

Comment:
@rmohr  fyi, this will need updating once #4710 lands

Reply:
@zhlhahaha #4710 landed. I think it will simplify your PR, but you have to rebase. You can remove all test-only RPM dependencies from the WORKSPACE file which you added and just extend `repo.yaml` and `hack/rpm-deps.sh`. You will have to keep the `libvirt-*` RPMs until #4703 lands.

Reply:
> @zhlhahaha #4710 landed. I think it will simplify your PR, but you have to rebase. You can remove all test-only RPM dependencies from the WORKSPACE file which you added and just extend `repo.yaml` and `hack/rpm-deps.sh`. You will have to keep the `libvirt-*` RPMs until #4703 lands.

Hi @rmohr I have tried new way to import rpms and it is quit easy to use, but I have a question here:
If I run `make rpm-deps`, some rpms for x86_64 or ppc64le would change in `WORKSPACE`. e.g.  libzstd rpm changed from "libzstd-1.4.5-4.fc32.x86_64.rpm" to "libzstd-1.4.7-1.fc32.x86_64.rpm" and "https://storage.googleapis.com/builddeps/...sha256..." in urls was deleted.

Do I need to run `make rpm-deps` to generate new version `WORKSPACE`, and put the new version `WORKSPACE` into this PR? Or just update `repo.yaml`, `rpm/BUILD.bazel` and `hack/rpm-deps.sh`.

Reply:
You update `repo.yaml` and `hack/rpm-deps.sh`. then run `make rpm-deps` and commit everything.

If new packages are available they will be used. In that case the shasum is different and the mirror url will also be removed. We have a job which will mirror the dependencies after the PR is merged. But you commit everything so that you as developer and ci see the same and final things.

Reply:
@zhlhahaha another pr landed which requires adaption: https://github.com/kubevirt/kubevirt/pull/4703 is now merged.

You can drop all custom libvirt-devel RPMs now and just add in `hack/rpm-deps.sh` a `libvirt-devel-arm64` target a `ldd` target in `rpm/BUILD.bazel` like for `x86_64`. Then you can use  a switch in BUILD.bazel to select the right headers and libraries for arm64.

That should be my last surprise which should adding multi-arch much easier.